source/_components/notify.notifyme.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: "Notify Me"
+description: "Instructions on how to add Alexa notification via NotifyMe to Home Assistant."
+date: 2018-11-27 00:00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: notifyme.png
+ha_category: Notifications
+ha_release: 0.83
+---
+
+NotifyMe is an Alexa Skill which allows you to send a notification to Alexa
+via a simple REST API, which will make your Echo devices to flash a yellow 
+ring to notify you.
+
+To use this notification platform in your installation, add the following
+to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+notify:
+  - name: NOTIFIER_NAME
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.